examples/cms-sanity/components/cover-image.js

```diff
@@ -0,0 +1,26 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
+      src={url}
+      alt={`Cover Image for ${title}`}
+      className={cn('shadow-small', {
+        'hover:shadow-medium transition-shadow duration-200': slug,
+      })}
```

Comment:
Can we use Sanity's image builder: https://www.sanity.io/docs/image-url

and:

- Use it to fetch 2000x1000 image
- Then set `width={2000}` and `height={1000}` to `img` tag so the layout doesn't jump around

Reply:
Updated